packages/terra-date-picker/translations/ar.json

```diff
@@ -1,5 +1,7 @@
 {
   "Terra.datePicker.dateFormat": "س‌س‌س‌س/ي‌ي/ش‌ش",
+  "Terra.datePicker.separator": "/",
   "Terra.datePicker.today": "اليوم",
-  "Terra.datePicker.openCalendar": "فتح التقويم"
-}
+  "Terra.datePicker.openCalendar": "فتح التقويم",
+  "Terra.datePicker.dateFormatOrder": "DMY"
+}
```

Comment:
We no longer support Arabic and we are missing some keys. Should we add to this file?

Reply:
cc: @neilpfeiffer

Reply:
Removed changes to "ar" and "fi-fi" translations here : https://github.com/cerner/terra-framework/pull/1208/commits/e4a82204c64a0416ab80e9efd7eb83eec54dfd83